packages/ember-runtime/lib/system/core_object.js

```diff
@@ -930,17 +907,4 @@ CoreObject.ClassMixin = ClassMixin;
 
 ClassMixin.apply(CoreObject);
 
-CoreObject.reopen({
-  didDefineProperty(proto, key, value) {
```

Comment:
I can leave this method, but given the rest of the changes here I do not see that it is needed...